src/Sulu/Bundle/AdminBundle/Metadata/FormMetadata/TagMetadata.php

```diff
@@ -42,4 +47,30 @@ public function setPriority(?int $priority): void
     {
         $this->priority = $priority;
     }
+
+    public function setAttributes(array $attributes): void
+    {
+        $this->attributes = $attributes;
+    }
+
+    public function getAttributes()
+    {
+        return $this->attributes;
+    }
+
+    public function getAttribute(string $name)
+    {
+        return $this->attributes[$name] ?? null;
+    }
+
+    public function matchAttributes(array $attributes): bool
```

Comment:
How about naming this `hasAttributes`? Would be clearer to me what this function does then. If you e.g. think about `preg_match` it does a little bit more than just returning a boolean.

Reply:
👍